### PR TITLE
schoolview for users

### PR DESCRIPTION
**Pull request by sameerav**
shows profile view from announcements and schooldetailview for each mentor from each page

### CODE REVIEW

File: sage-ios/SAGE/SAGE/AnnouncementsDetailView.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Foundation
 class AnnouncementsDetailView: UIView {
     var scrollView = UIScrollView()
     let announcementUserImg = ProfileImageView()
+    let announcementUserButton = UIButton()
```
Comment (amillman) [inline sage-ios/SAGE/SAGE/AnnouncementsDetailView.swift]: noooooooo dont  do this. you dont need a button, you can add a TapGestureRecognizer on the announcementUserImg instead

File: sage-ios/SAGE/SAGE/SchoolDetailViewController.swift [hunk 2]
```diff
@@ -34,6 +36,31 @@ class SchoolDetailViewController: UITableViewController {
...
+                for var index = 0; index < students.count; ++index {
+                    let student = students[index]
+                    if student.id == newUser.id {
+                        reload = true
+                        self.school!.students![index] = newUser
+                    }
+                }
+            }
+            
+            if reload {
```
Comment (amillman) [inline sage-ios/SAGE/SAGE/SchoolDetailViewController.swift]: dont need this reload thing instead you can reload a specific cell at an indexPath, which you will be able to get by the index of the user in the studentsArray

File: sage-ios/SAGE/SAGE/SchoolDetailViewController.swift [hunk 3]
```diff
@@ -70,6 +103,13 @@ class SchoolDetailViewController: UITableViewController {
             marker.map = self.schoolDetailHeaderView.mapView
             self.schoolDetailHeaderView.mapView.moveCamera(GMSCameraUpdate.setTarget(coordinate))
         }
+        
+        self.schoolDetailHeaderView.mapView.hidden = true
```
Comment (amillman) [inline sage-ios/SAGE/SAGE/SchoolDetailViewController.swift:102]: why is this change in the same PR....

Comment (sameerav) [inline sage-ios/SAGE/SAGE/SchoolDetailViewController.swift:102]: sry

**Pull request merged**